python/ray/experimental/placement_group.py

```diff
@@ -0,0 +1,26 @@
+import ray
+from typing import (List, Dict)
+
+
+def placement_group(bundles: List[Dict[str, float]], strategy: str = "PACK",
+                    name: str = None):
+    """
+    placement_group.
```

Comment:
```suggestion
    Create a placement group.
```

(please also remove the newline, so it matches the style of other docstrings)